### DISCUSSION TITLE
آموزش استفاده از کاستوم کانفیگ در مرزبان

### DISCUSSION THREAD

**Discussion by Kiya6955** [Gozargah/Marzban#1099] [Show and tell]
### امروز با آموزش استفاده از کاستوم کانفیگ داخل پنل مرزبان در خدمتتونم

- در ابتدا با استفاده از nano وارد env میشیم.
```
nano /opt/marzban/.env
```
- مقادیر زیر رو اضافه میکنیم.

برای کلاینت های V2rayN,V2rayNG,Streisand :
```
USE_CUSTOM_JSON_FOR_V2RAYN=True
USE_CUSTOM_JSON_FOR_V2RAYNG=True
USE_CUSTOM_JSON_FOR_STREISAND=True
```
برای تمامی کلاینت ها :
```
USE_CUSTOM_JSON_DEFAULT=True
```
- سپس پنل رو با دستور زیر ریستارت میکنیم تا تغییرات اعمال بشه
```
marzban restart
```
با موفقیت کاستوم کانفیگ پنلتون فعال شد و الان میتونید از داخل Host Settings فرگمنت ماکس و یا UA رو تنظیم کنید.

همچنین میتونید از [تمپلیتWhyMan1 ](https://github.com/WhyMan1/marzban-template/tree/master?tab=readme-ov-file)برای تنظیم روتینگ دلخواه خودتون سمت کلاینت ها استفاده کنید.

#CustomConfig

**Comment by iamtheted** [comment]
دوست عزیزم، ممنونم از وقتی که گذاشتی. 💙

**Comment by plasticgholam** [comment]
برای استفاده از تمپلت یوزر ایجنت رندم هم باید تمپلتش رو از [اینجا](https://github.com/Gozargah/Marzban/tree/dev/app/templates/user_agent) دانلود کرد و در فولدرش قرار داد و مقادیر زیر رو در .env اضافه کرد:

USER_AGENT_TEMPLATE = True
GRPC_USER_AGENT_TEMPLATE = True

**Comment by farshadl** [comment]
ممنون بابت آموزشتون. خیلی کاربردی بود
یک سوال دارم. کاربرد UA و یوزر ایجنت رندم که در بالا گفتید چیه؟

**Reply by Kiya6955** [reply]
https://bmbgk.ir/?q=What+is+UserAgent

**Comment by JavadWolf-af** [comment]
سلام مهندس وقتی تبدیل میکنی به کاستوم لیشت کانفیگها بهم میریزه میدونی باید چیکار کنیم تا درست شه؟منظورم مرتبشون کنیم

**Reply by Kiya6955** [reply]
درود در ریلیز مرزبان بعدی رفع میشه.
